docs/src/main/asciidoc/reactive-routes.adoc

```diff
@@ -101,6 +101,19 @@ public void blocking(RoutingContext rc) {
 }
 ----
 
+[NOTE]
+====
+Alternatively, you can use `@RunOnWorkerThread` and omit the `type = Route.HandlerType.BLOCKING`:
```

Comment:
I think that we should be clear that the handler type is ignored if the annotation is present.

Reply:
Is it worth deprecating the `blocking` attribute of `@Route` in favor of the specific annotation?

Reply:
I'm not quite sure. First of all the original idea was to avoid the "annotation hell" and keep all relevant info in one annotation. Also it's not a "blocking" attribute but a type of the handler and we still need the one for failures.

Reply:
That's fine, was just curious

Reply:
Added a note in the documentation.